crm_claim_rma/models/account_invoice.py

```diff
@@ -16,6 +16,10 @@ class AccountInvoice(models.Model):
 
     claim_id = fields.Many2one('crm.claim', string='Claim')
 
+    partner_id = fields.Many2one('res.partner', index=True)
+
+    commercial_partner_id = fields.Many2one('res.partner', index=True)
```

Comment:
Do we really want to force the indexing of these fields when the module is installed ? :thinking: 
I'm not sure this OCA module is the right place to do it, as it can easily be done in a specific module for projects in need of this....